### DISCUSSION TITLE
ImageGlass won't remember window size, opening images in increasingly larger sizes each time opened until it reaches the full width of both of my dual monitors.

### DISCUSSION THREAD

**Discussion by JayNeely** [d2phap/ImageGlass#2011] [Q&A]
I'm having an issue where ImageGlass won't remember the window size or zoom level, opening images in a larger window than what I last had it sized to and closed it at, every time I open an image, despite being in Scale-to-Fit mode.

If I open a 2802 x 1440 image and resize the window to a little notecard sized window near the left edge of my second (left-side) monitor with the image at 32.44% zoom, then close and reopen the same image, it now opens at 49.07% zoom, about a half a sheet of paper window size. If I close and reopen again, it now opens at 74.02% zoom, about a letter-sized piece of paper window size. If I close and reopen again, it opens at 111.42% zoom, now overflowing off one monitor and onto the next. Repeating, 145.54%, window now at the full width of both monitors. Only once I open it again does it stop growing, having reached the right edge of my primary (right-side) monitor.

If I open a smaller 600 x 331 image and repeat the process, the same thing happens. If I change it to AutoZoom mode instead of Scale to fit, the zoom level stops increasing once it reaches 100% but the window around the image still keeps getting larger and larger with more whitespace around the image on every close and reopen.

I'm running 9.1.8.723 (64-bit) on Windows 11, with a dual-screen setup (vertical monitor next to a horizontal monitor). I've tried a bunch of web-suggested remedies for window sizes not being remembered: closing ImageGlass by double-clicking the top-left icon, right-clicking the top-left icon and using the size tool, and unchecking and rechecking the "Remember window locations based on monitor connection" option in Windows Settings > System> Display > Multiple Displays. 

I'm not experiencing this issue with any other programs (Windows Explorer, browser windows), just ImageGlass. I've tried uninstalling and reinstalling ImageGlass. I've tried resetting settings. I'm not sure what else to try. 

This is making ImageGlass unusable for me, as it essentially requires me to resize the window every time I open an image. Help?

**Comment by d2phap** [comment]
Hi @JayNeely 

Can you check if you accidentally enabled "Window Fit" mode (Main Menu > Window Fit)?
You can try resetting the app settings if it helps: open Main Menu > Settings, click the "Reset settings" link at the bottom left corner

**Reply by JayNeely** [reply]
Hey @d2phap 👋 I checked; Window Fit is not enabled. I enabled and disabled just to make sure. Resetting settings didn't fix things, but it did illuminate the issue: **it's something to do with how the app interacts with the second monitor specifically.** 

After resetting settings, the default image opened up on my main screen and I closed it, then opened another image, closed it, and opened it again -- the issue did _not_ reoccur. This was confusing because I had reset settings before and it didn't fix it, so I kept testing it moving the image into different positions, closing, and reopening. The window enlargement effect only kicks in when the image was last closed on the second monitor.
